local-cluster/tests/local_cluster.rs

```diff
@@ -5710,20 +5711,19 @@ fn test_randomly_mixed_block_verification_methods_between_bootstrap_and_not() {
          info",
     );
 
-    let num_nodes = 2;
+    let num_nodes = BlockVerificationMethod::COUNT;
```

Comment:
.... so that someone can challenge my baby as a 3rd method of the block verification. lol